application.py

```diff
@@ -6,7 +6,8 @@
 
 application = create_app(os.getenv('DM_ENVIRONMENT') or 'development')
 manager = Manager(application)
-manager.add_command("runserver", Server(port=5002))
+manager.add_command("runserver", Server(
+    os.getenv('DM_HOST') or '127.0.0.1', port=5002))
```

Comment:
Should the value for AWS be the default then? I based it on the `DM_ENVIRONMENT` variable which defaults to a development setting.

I guess I need to look into changing the AWS configuration before this can be merged yeah?